infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/usecases/TokenizeEventPayloadFieldsUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.simprints.infra.eventsync.event.usecases
+
+import com.simprints.core.domain.tokenization.TokenizableString
+import com.simprints.infra.config.store.models.Project
+import com.simprints.infra.config.store.tokenization.TokenizationProcessor
+import javax.inject.Inject
+import com.simprints.infra.events.event.domain.models.Event
+
+internal class TokenizeEventPayloadFieldsUseCase @Inject constructor(
+    private val tokenizationProcessor: TokenizationProcessor
+) {
+    operator fun invoke(event: Event, project: Project): Event {
+        val tokenizedFields = event.getTokenizedFields().mapValues { entry ->
```

Comment:
Naming is a little confusing here, first we `getTokenizedFields()` and then check whether they are indeed tokenized and do tokenize some of them :D Would it be more appropriate to rename Event function to `getTokenizableFields()`? After all it returns the parent `TokenizableString`